MAUI/Maps/migration.md

```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
+   <th>Xamarin SfMaps</th>
+   <th>.NET MAUI SfMaps</th>
+</tr>
+<tr>
+<tr>
+   <td>
+       {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'MapCircle'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+       {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Center'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Radius'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+       {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Stroke'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeThickness'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Fill'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
+
+<table>
+<tr>
+<th>Xamarin SfMaps</th>
+<th>.NET MAUI SfMaps</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+{% endhighlight %}
+
+{% endtabs %}
+</td>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+ <map:SfMaps>
+    <map:SfMaps.Layer>
+        <map:MapShapeLayer ShapesSource="https://cdn.syncfusion.com/maps/map-data/india.json">
+            <map:MapShapeLayer.Sublayers>
+                <map:MapCircleLayer>
+                    <map:MapCircleLayer.Circles>
+                        <map:MapCircle Radius="10" 
+                                       Fill="LightGreen" 
+                                       Stroke="Green" 
+                                       StrokeThickness="2">
+                            <map:MapCircle.Center>
+                                <map:MapLatLng Latitude="74.1240"
+                                               Longitude="15.2993"></map:MapLatLng>
+                            </map:MapCircle.Center>
+                        </map:MapCircle>
+                    </map:MapCircleLayer.Circles>
+                </map:MapCircleLayer>
+            </map:MapShapeLayer.Sublayers>
+        </map:MapShapeLayer>
+    </map:SfMaps.Layer>
+</map:SfMaps>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+public MainPage()
+{
+    InitializeComponent();
+    SfMaps maps = new SfMaps();
+    MapShapeLayer layer = new MapShapeLayer();
+    layer.ShapesSource = MapSource.FromUri(new Uri("https://cdn.syncfusion.com/maps/map-data/india.json"));
+    MapCircleLayer circleLayer = new MapCircleLayer();
+    MapCircle circle1 = new MapCircle();
+    circle1.Center = new MapLatLng(74.1240, 15.2993);
+    circle1.Radius = 10;
+    circle1.Fill = Colors.LightGreen;
+    circle1.Stroke = Colors.Green;
+    circle1.StrokeThickness = 3;
+    circleLayer.Circles.Add(circle1);
+    maps.Layer = layer;
+    this.Content = maps;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
 ## Upcoming features in .NET MAUI
 
 * Support for imagery layer such as Bing Maps, OpenStreetMap, etc.
```

Comment:
* Support for imagery layers such as Bing Maps, OpenStreetMap, and more.

Reply:
Modified